lib/tilebag.rb

```diff
@@ -0,0 +1,31 @@
+require 'awesome_print'
+require 'pry'
+require_relative 'player'
+
+module Scrabble
+  class TileBag
+    attr_accessor :tilebag
+
+    def initialize
+      @tilebag = { "a" => 9, "b" => 2, "c" => 2, "d" => 4, "e" => 12, "f" => 2, "g" => 3, "h" => 2, "i" => 9, "j" => 1, "k" => 1, "l" => 4, "m" => 2, "n" => 6, "o" => 8, "p" => 2, "q" => 1, "r" => 6, "s" => 4, "t" => 6, "u" => 4, "v" => 2, "w" => 2, "x" => 1, "y" => 2, "z" => 1 }
```

Comment:
You're giving each letter an equal chance of being drawn with this data structure, but there are **12** "E" tiles and only **1** "J".